src/libraries/System.Globalization/tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -90,7 +90,8 @@ public void DisplayName(string name, string expected)
             }
         }
 
-        [Theory]
+        // RegionInfo.NativeName is not supported by Browser's ICU
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
```

Comment:
Check the actual values

Reply:
the test actually works with current version of ICU.
I am going to strip more from ICU (https://github.com/dotnet/icu/pull/5) but that can be update in a separate PR